### PR TITLE
BETA: Update oxmc.is-a.dev

### PR DESCRIPTION
**Pull request by oxmc**
Updated `oxmc.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/oxmc.json [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
-        },
-        "record": {
-            "CNAME": "oxmc.github.io"
-        }
+    "owner": {
+        "username": "oxmc",
+        "email": "oxmc7769.mail@gmail.com"
+    },
+    "record": {
+            "MX": "mx2.forwardemail.net"
```
Comment (WilliamDavidHarrison) [inline domains/oxmc.json:7]:
```suggestion
        "MX": ["mx1.forwardemail.net", "mx2.forwardemail.net"]
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/oxmc.json (hunk 1 above)
Comment (oxmc) [reply domains/oxmc.json:7]: Should i also re-add my CNAME?

**Review by wdhdev — CHANGES_REQUESTED** [review]
You can't have a CNAME and MX record together.

**Comment by oxmc** [comment]
> 

Oh, i never knew that, do i make a subdomain then for emails?

**Comment by oxmc** [comment]
Actually, is that even possible @WilliamDavidHarrison ?

**Comment by WilliamDavidHarrison** [comment]
> Oh, i never knew that, do i make a subdomain then for emails?

You just need to use GitHub Pages' A records. You can find them on the GitHub Pages documentation.

**Comment by oxmc** [comment]
> > Oh, i never knew that, do i make a subdomain then for emails?
> 
> You just need to use GitHub Pages' A records. You can find them on the GitHub Pages documentation.

Ah, alright, thank you for helping.

**Comment by oxmc** [comment]
Alright, replaced CNAME with A record.

**Comment by oxmc** [comment]
Added my final changes @WilliamDavidHarrison

**Review by wdhdev — APPROVED** [review]

**Pull request merged**